tests/unit/general/bin-update-ips-subnet.bats

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bats
+
+# bats file_tags=static,general,bin:update_ips
+
+setup_file() {
+  load "../../bats.lib.bash"
+  load_common "env.bash"
+  load_common "gpg.bash"
+  load_common "yq.bash"
+
+  gpg.setup
+  env.setup
+
+  env.init openstack capi dev
```

Comment:
It would be nice to test the fallback in a non-openstack and/or non-capi environment as well.

Reply:
That should be what all the existing tests in https://github.com/elastisys/compliantkubernetes-apps/blob/e7e05d51347bc3cd407fc53542fbdc1cb5058bac/tests/unit/general/bin-update-ips.bats does. No?

Reply:
Ah, never mind then.